apps/laboratory/src/utils/CommonUtils.ts

```diff
@@ -0,0 +1,15 @@
+export function parseJSON(str: string) {
+  try {
+    return JSON.parse(str)
+  } catch (error) {
+    return { error: `Invalid JSON: ${error}` }
+  }
+}
+
+export function bigIntReplacer(_key: string, value: bigint) {
```

Comment:
Seems like `value` already could be `bigint`, why do we need conditional return in the body? Is it unknown or non-bigint in the place where this function used?

Reply:
It need to be change to any, and if it type is bigint, then value.toString. I will do the change.

Reply:
@KannuSingh I'm not sure if it should be `any`, don't we know possible types of the parameter we pass to this function? If it's not known, we could use `unknown`, otherwise we should keep as strict as possible.

Reply:
it is used in JSON.stringify(batchCallsStatus, bigIntReplacer).


```
(method) JSON.stringify(value: any, replacer?: ((this: any, key: string, value: any) => any) | undefined, space?: string | number | undefined): string 
```